### DISCUSSION TITLE
Error in creating 2 digit by 2 digit rangefinder array using <replicate/>

### DISCUSSION THREAD

**Discussion by yzwfromk** [google-deepmind/mujoco#2088] [Asking for Help]
### Intro

Hi!

I am a graduate student at Georgia Tech, I use MuJoCo for my research on bipedal robots and simulating LiDAR and Depth camera.

### My setup

Ubuntu 20.04.6 LTS, 64 bit
Mujoco Python 3.2.0

### My question

Recently I tried to mimic LiDAR using replicate for rangefinder sensors. However, I found a weird issue while using replicate as below:
```
<mujoco model='replicate error'>
  <compiler angle='degree' eulerseq='xyz'/>
  
  <worldbody>
    <geom name='ground' type='plane' size='0 0 1'/>
    <light diffuse="1 1 1" pos="0 0 3" dir="0 0 -1"/>
    
    <body name='base' pos='0 0 1' euler='0 0 0'>
      <geom type='cylinder' size='0.05 0.0325' pos='0 0 0' euler='-0 0 -0'/>
      <replicate count="91" offset="0 0 0" euler="0 0 1">
        <replicate count="20" offset="0 0 0" euler="0 1 0">
          <site name="rangefinder_array" pos="0 0 0" zaxis="1 0 -1"/>
        </replicate>
      </replicate>
    </body>
  </worldbody>

  <sensor>
    <rangefinder  name='LiDAR' site='rangefinder_array'/>
  </sensor>

</mujoco>

```

When load the XML file, it gives me error saying:
`
XML Error: Error: repeated name 'LiDAR0000' in sensor
Element 'replicate', line 11
`

However, If I change the number '20' to '9' or '111' or whatever number other than 2 digits number, it works without any error.
So the thing is, when I create 2 digits number-by-1-digit number (or 3 digit number), it works. However, for 2 digits number-by-2 digits number, doesn't work.

Could you help me with this?

### Confirmations

- [X] I searched the [latest documentation](https://mujoco.readthedocs.io/en/latest/overview.html) thoroughly before posting.
- [x] I searched previous [Issues](https://github.com/google-deepmind/mujoco/issues) and [Discussions](https://github.com/google-deepmind/mujoco/discussions), I am certain this has not been raised before.

**Comment by yuvaltassa** [comment]
This is great, except that it should be a bug report (an issue), rather than in Asking for help (a discussion).
The main difference is that bugs can be assigned to people while discussions cannot.
You can do that if you want (and then close this discussion), but this is fine; I'm mostly telling you for future reference 🙂

**Reply by yzwfromk** [reply]
ok let me do it again, thanks for letting me know
